### PR TITLE
[Refactor] move and change localization key of battler-tags

### PR DESCRIPTION
**Pull request by EnochG1**
## What are the changes?
There are no user-facing changes.

## Why am I doing these changes?
- in locales/{language}/battle.ts, there are battler-tags messages which have long and same localization key so I want to separate it.
- Stylistic consistency.

## What did change?
- move battler tags messages -> from locales/{language}/battle.ts to locales/{language}/battler-tags.ts
- change localization key -> from battlerTags{Something} to {something}
- example)
```Typescript
i18next.t("battle:battlerTagsNightmareOnAdd", { pokemonNameWithAffix: getPokemonNameWithAffix(pokemon) }) // before
i18next.t("battlerTags:nightmareOnAdd", { pokemonNameWithAffix: getPokemonNameWithAffix(pokemon) }) // after

"battlerTagsNightmareOnAdd": "{{pokemonNameWithAffix}} began\nhaving a Nightmare!", // before in battle.ts
"nightmareOnAdd": "{{pokemonNameWithAffix}} began\nhaving a Nightmare!", // after in battler-tags.ts
```

### Screenshots/Videos
N/A

## How to test the changes?
Manually (Using overrides.ts)

## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [ ] Are all unit tests still passing? (`npm run test`)
- [ ] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/ca_ES/battler-tags.ts [hunk 1]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "nightmares",
   "ingrainDesc": "roots",
   "drowsyDesc": "drowsiness",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}} must\nrecharge!",
```
Comment (DayKev) [inline src/locales/ca_ES/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}} must\nrecharge!",
```

File: src/locales/de/battler-tags.ts [hunk 2]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "Nachtmahr",
   "ingrainDesc": "Verwurzlung",
   "drowsyDesc": "Schläfrigkeit",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}} kann sich wegen des Rückstoßes durch den Angriff nicht bewegen!",
```
Comment (DayKev) [inline src/locales/de/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}} kann sich wegen des Rückstoßes durch den Angriff nicht bewegen!",
```

File: src/locales/en/battler-tags.ts [hunk 3]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "nightmares",
   "ingrainDesc": "roots",
   "drowsyDesc": "drowsiness",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}} must\nrecharge!",
```
Comment (DayKev) [inline src/locales/en/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}} must\nrecharge!",
```

File: src/locales/es/battler-tags.ts [hunk 4]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "nightmares",
   "ingrainDesc": "roots",
   "drowsyDesc": "drowsiness",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}} must\nrecharge!",
```
Comment (DayKev) [inline src/locales/es/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}} must\nrecharge!",
```

File: src/locales/fr/battler-tags.ts [hunk 5]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "les cauchemars",
   "ingrainDesc": "l’enracinement",
   "drowsyDesc": "la somnolence",
+
+  "rechargingLapse": "Le contrecoup empêche {{pokemonNameWithAffix}}\n de bouger !",
```
Comment (DayKev) [inline src/locales/fr/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "Le contrecoup empêche {{pokemonNameWithAffix}}\n de bouger !",
```

File: src/locales/it/battler-tags.ts [hunk 6]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "nightmares",
   "ingrainDesc": "roots",
   "drowsyDesc": "drowsiness",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}} deve\nricaricarsi!",
```
Comment (DayKev) [inline src/locales/it/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}} deve\nricaricarsi!",
```

File: src/locales/ja/battler-tags.ts [hunk 7]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "nightmares",
   "ingrainDesc": "roots",
   "drowsyDesc": "drowsiness",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}}は　攻撃の　反動で　動けない！",
```
Comment (DayKev) [inline src/locales/ja/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}}は　攻撃の　反動で　動けない！",
```

File: src/locales/ko/battler-tags.ts [hunk 8]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "악몽",
   "ingrainDesc": "뿌리",
   "drowsyDesc": "졸음",
+
+  "rechargingLapse": "공격의 반동으로\n{{pokemonNameWithAffix}}[[는]] 움직일 수 없다!",
```
Comment (DayKev) [inline src/locales/ko/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "공격의 반동으로\n{{pokemonNameWithAffix}}[[는]] 움직일 수 없다!",
```

File: src/locales/pt_BR/battler-tags.ts [hunk 9]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "os pesadelos",
   "ingrainDesc": "o enraizamento",
   "drowsyDesc": "a sonolência",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}} precisa\nrecarregar!",
```
Comment (DayKev) [inline src/locales/pt_BR/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}} precisa\nrecarregar!",
```

File: src/locales/zh_CN/battler-tags.ts [hunk 10]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "恶梦",
   "ingrainDesc": "扎根",
   "drowsyDesc": "瞌睡",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}}\n因攻击的反作用力而无法动弹！",
```
Comment (DayKev) [inline src/locales/zh_CN/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}}\n因攻击的反作用力而无法动弹！",
```

File: src/locales/zh_TW/battler-tags.ts [hunk 11]
```diff
@@ -9,4 +9,66 @@ export const battlerTags: SimpleTranslationEntries = {
   "nightmareDesc": "惡夢",
   "ingrainDesc": "扎根",
   "drowsyDesc": "瞌睡",
+
+  "rechargingLapse": "{{pokemonNameWithAffix}}因攻擊的反作用力而無法動彈！",
```
Comment (DayKev) [inline src/locales/zh_TW/battler-tags.ts:13]:
```suggestion
  "rechargingLapse": "{{pokemonNameWithAffix}}因攻擊的反作用力而無法動彈！",
```

**Review by DayKev — CHANGES_REQUESTED** [review]
Minor newline issue.

**Review by ben-lear — APPROVED** [review]
lgtm

**Review by DayKev — APPROVED** [review]

**Pull request merged**